app/_src/quickstart/universal-demo.md

```diff
@@ -0,0 +1,258 @@
+  npm start --prefix=app/
+  ```
+
+1. In a browser, go to [127.0.0.1:5000](http://127.0.0.1:5000) and increment the counter.
+
+## Explore the GUI
+
+You can view the sidecar proxies that are connected to the {{site.mesh_product_name}} control plane:
+
+{% tabs usage useUrlFragment=false %}
+{% tab usage GUI (Read-Only) %}
+
+{{site.mesh_product_name}} ships with a **read-only** GUI that you can use to retrieve {{site.mesh_product_name}} resources. By default the GUI listens on the API port and defaults to `:5681/gui`. 
+
+You can navigate to [`127.0.0.1:5681/meshes/default/dataplanes`](http://127.0.0.1:5681/meshes/default/dataplanes) to see the connected dataplanes.
+
+{% endtab %}
+{% tab usage HTTP API (Read/Write) %}
+
+{{site.mesh_product_name}} ships with a **read-only** HTTP API that you can use to retrieve {{site.mesh_product_name}} resources. 
+
+By default the HTTP API listens on port `5681`. 
+
+Navigate to [`127.0.0.1:5681/meshes/default/dataplanes`](http://127.0.0.1:5681/meshes/default/dataplanes) to see the connected dataplanes.
+
+{% endtab %}
+{% tab usage kumactl (Read/Write) %}
+
+You can use the `kumactl` CLI to perform **read-only** operations on {{site.mesh_product_name}} resources. The `kumactl` binary is a client to the {{site.mesh_product_name}} HTTP API, you will need to first port-forward the API service with:
+
+Run `kumactl`, for example:
+
+```sh
+kumactl get dataplanes
+# MESH      NAME                                              TAGS
+# default   kuma-demo-app-68758d8d5d-dddvg.kuma-demo          app=kuma-demo-demo-app env=prod pod-template-hash=68758d8d5d protocol=http service=demo-app_kuma-demo_svc_5000 version=v8
+# default   redis-master-657c58c859-5wkb4.kuma-demo           app=redis pod-template-hash=657c58c859 protocol=tcp role=master service=redis_kuma-demo_svc_6379 tier=backend
+```
+
+You can configure `kumactl` to point to any zone `kuma-cp` instance by running:
+
+```sh
+kumactl config control-planes add --name=XYZ --address=http://{address-to-kuma}:5681
+```
+{% endtab %}
+{% endtabs %}
+
+## Introduction to zero-trust security
+
+By default, the network is insecure and not encrypted. We can change this with {{site.mesh_product_name}} by enabling the [Mutual TLS](/docs/{{ page.version }}/policies/mutual-tls/) policy to provision a Certificate Authority (CA) that will automatically assign TLS certificates to our services (more specifically to the injected data plane proxies running alongside the services).
+
+Before enabling [Mutual TLS](/docs/{{ page.version }}/policies/mutual-tls/) (mTLS) in your mesh, you need to create a `MeshTrafficPermission` policy that allows traffic between your applications.
+
+{% warning %}
+If you enable [mTLS](/docs/{{ page.version }}/policies/mutual-tls/) without a `MeshTrafficPermission` policy, all traffic between your applications will be blocked. 
+{% endwarning %}
+
+1. To create a `MeshTrafficPermission` policy that allows all traffic, do the following:
```

Comment:
Numbering is wrong:

<img width="558" alt="image" src="https://github.com/kumahq/kuma-website/assets/2753650/22fd80b5-7289-4252-89db-363647aef411">

```suggestion
2. To create a `MeshTrafficPermission` policy that allows all traffic, do the following:
```